### ISSUE TITLE
overflow issues with java.io.PrintStream.write(PrintStream.java:480)

### ISSUE THREAD

**Issue by nicfel** [nicfel/CoupledMCMC#4]
Getting overflow errors with java.io.PrintStream.write(PrintStream.java:480). Could be an issue with copying the loggers the wrong way

**Comment by nicfel** [comment]
Get the following out of memory error:

```
Exception in thread "main" java.lang.OutOfMemoryError: Java heap space
	at java.nio.HeapByteBuffer.<init>(HeapByteBuffer.java:57)
	at java.nio.ByteBuffer.allocate(ByteBuffer.java:335)
	at sun.nio.cs.StreamEncoder.<init>(StreamEncoder.java:195)
	at sun.nio.cs.StreamEncoder.<init>(StreamEncoder.java:175)
	at sun.nio.cs.StreamEncoder.forOutputStreamWriter(StreamEncoder.java:59)
	at java.io.OutputStreamWriter.<init>(OutputStreamWriter.java:111)
	at java.io.PrintStream.<init>(PrintStream.java:104)
	at java.io.PrintStream.<init>(PrintStream.java:151)
	at java.io.PrintStream.<init>(PrintStream.java:135)
	at beast.coupledMCMC.CoupledMCMC.swapLoggers(Unknown Source)
	at beast.coupledMCMC.CoupledMCMC.runPrescheduled(Unknown Source)
	at beast.coupledMCMC.CoupledMCMC.run(Unknown Source)
	at beast.app.BeastMCMC.run(Unknown Source)
	at beast.app.beastapp.BeastMain.<init>(Unknown Source)
	at beast.app.beastapp.BeastMain.main(Unknown Source)
java.lang.UNIXProcess$ProcessPipeInputStream@9807454
```

**Comment by nicfel** [comment]
solved in v0.1.2
